### PR TITLE
[NUI] Introduce Notification

### PR DESCRIPTION
**Pull request by rabbitfor**
Signed-off-by: Jiyun Yang <ji.yang@samsung.com>

### API Changes ###

 - ACR:

### CODE REVIEW

File: src/Tizen.NUI.Components/Controls/Notification.cs [hunk 1]
```diff
@@ -0,0 +1,394 @@
...
+        /// The return value means delay dismissal in ms (0 means dismiss directly).
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public delegate uint OnDismissDelegate(Notification notification);
+
+        private enum NotificationState
+        {
+            Ready,
+            Post,
+            DelayingOnDismiss,
```
Comment (hyunjushin) [inline src/Tizen.NUI.Components/Controls/Notification.cs:80]: What is the meaning of DelayingOnDismiss?

File: src/Tizen.NUI.Components/Controls/Notification.cs [hunk 2]
```diff
@@ -0,0 +1,394 @@
...
+
+            return this;
+        }
+
+        /// <summary>
+        /// Sets a delegate to be called after the notification dismiss.
+        /// </summary>
+        /// <param name="onDismiss">The delegate to be called.</param>
+        /// <returns>The current Notification instance.</returns>
+        /// <remarks>The return value of the delegate means delay dismissal in ms (0 means dismiss directly).</remarks>
```
Comment (hyunjushin) [inline src/Tizen.NUI.Components/Controls/Notification.cs:272]: please check the return type.

File: src/Tizen.NUI.Components/Controls/Notification.cs [hunk 3]
```diff
@@ -0,0 +1,394 @@
...
+        public Notification SetLevel(NotificationLevel level)
+        {
+            this.level = level;
+
+            if (state == NotificationState.Post || state == NotificationState.DelayingOnDismiss)
+            {
+                ApplyLevel(level);
+            }
+
+            return this;
```
Comment (sparrow74) [inline src/Tizen.NUI.Components/Controls/Notification.cs:204]: Why Notification instance is returned after setting level?

File: src/Tizen.NUI.Components/Controls/Notification.cs (hunk 1 above)
Comment (rabbitfor) [reply src/Tizen.NUI.Components/Controls/Notification.cs:80]: Please visit [here](https://github.com/dalihub/nui-demo/tree/master/wearable-samples/Notification/FullsizeNotification#define-ondismiss-behavior) to see onDismiss return value usage.

User can delay dismiss by defining onDismiss delegate.

File: src/Tizen.NUI.Components/Controls/Notification.cs (hunk 3 above)
Comment (rabbitfor) [reply src/Tizen.NUI.Components/Controls/Notification.cs:204]: To enable calling API continuously.
```C#
new Notification(contentView)
    .SetDismissOnTouch(true)
    .SetOnPostDelegate(OnNotificationPost)
    .SetOnDismissDelegate(OnNotificationDismiss)
    .Post();
```

**Comment by rabbitfor** [comment]
Demo : https://github.com/dalihub/nui-demo/tree/master/wearable-samples/Notification/FullsizeNotification

**Pull request merged**